Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/coreos/tectonic-console-builder:v18 AS build
+FROM quay.io/coreos/tectonic-console-builder:v19 AS build
```

Comment:
Just gonna note that this is:
https://quay.io/repository/coreos/tectonic-console-builder?tab=tags
Which we are manually building & pushing per convo, not CI.